### PR TITLE
aircrack-ng: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by Magrid0 — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.es/common/aircrack-ng.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# aircrack-ng
+
+> Crackea claves WEP y WPA/WPA2 desde handshake en paquetes capturados.
+> Parte de la suite de software de red Aircrack-ng.
+> Más información: <https://www.aircrack-ng.org/doku.php?id=aircrack-ng>.
+
+- Crackea la clave desde el archivo de captura usando [w]ordlist:
+
+`aircrack-ng -w {{ruta/a/lista.txt}} {{ruta/a/captura.cap}}`
```
Comment (blueskyson) [inline pages.es/common/aircrack-ng.md:9]: Reference: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md
```suggestion
`aircrack-ng -w {{ruta/al/lista.txt}} {{ruta/al/captura.cap}}`
```

File: pages.es/common/aircrack-ng.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- Descifra la clave del archivo de captura utilizando [w]ordlist y el [e]ssid del punto de acceso:
+
+`aircrack-ng -w {{ruta/a/lista.txt}} -e {{essid}} {{ruta/a/captura.cap}}`
```
Comment (blueskyson) [inline pages.es/common/aircrack-ng.md:13]:
```suggestion
`aircrack-ng -w {{ruta/al/lista.txt}} -e {{essid}} {{ruta/al/captura.cap}}`
```

File: pages.es/common/aircrack-ng.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+- Descifra la clave del archivo de captura utilizando [w]ordlist y la dirección MAC del punto de acceso:
+
+`aircrack-ng -w {{ruta/a/lista.txt}} --bssid {{mac}} {{ruta/a/captura.cap}}`
```
Comment (blueskyson) [inline pages.es/common/aircrack-ng.md:17]:
```suggestion
`aircrack-ng -w {{ruta/al/lista.txt}} --bssid {{mac}} {{ruta/al/captura.cap}}`
```

**Review by blueskyson — APPROVED** [review]

**Pull request merged**